prometheus-exporters/vrops-exporter/alerts/host.alerts

```diff
@@ -199,28 +199,84 @@ groups:
       description: "vSphere High Availability (HA) has detected a possible host failure for `{{ $labels.hostsystem }}`. ({{ $labels.vcenter }})"
       summary: "vSphere High Availability (HA) has detected a possible host failure for `{{ $labels.hostsystem }}`. ({{ $labels.vcenter }})."
 
-  - alert: HostMemoryUtilizationHigh
+  - alert: APodMemoryUtilizationHigh
     expr: vrops_hostsystem_memory_usage_percentage{vccluster=~".*management.*"} > 90
     for: 30m
     labels:
       severity: info
       tier: vmware
       service: compute
       context: "{{ $labels.hostsystem }} memory"
-      meta: "Memory utilization of host {{ $labels.hostsystem }} is above 90%. ({{ $labels.vcenter }}, {{ $labels.vccluster }})"
+      meta: "Memory utilization of host {{ $labels.hostsystem }} is above 90%. ({{ $labels.vcenter }})"
```

Comment:
Is the cluster not relevant anymore?

Reply:
I would like to keep it as clean as possible. The cluster is also known throughout the `hostsystem` label.